📝 doc(methodologie): Ajout d'un paragraphe détaillant la méthodologie d'analyse (#175)
cnumr/EcoIndex: content/en/how-it-works/_65_methodoloie.md

+++
id = "methodologie"
weight = 65
+++

{{% info_2_columns %}}

{{% info_2_columns_col1 %}}

## Méthodologie d'analyse

{{% /info_2_columns_col1 %}}

{{% info_2_columns_col2 %}}

The ecoindex.fr website uses the [ecoindex_api](https://github.com/cnumr/ecoindex_api) API to run page analyses. The ecoindex_api project embeds a [chrome driver for selenium](https://github.com/ultrafunkamsterdam/undetected-chromedriver).

> When we launch an analysis, we will therefore load the requested page in a real Chrome browser and play the following scenario:

Reviewer: ```suggestion
When an analysis is launched, the requested page is loaded in a real Chrome browser and play the following scenario:
```
